lib/vnet/opensshconfig.go

```diff
@@ -261,3 +264,64 @@ type configFileTemplateInput struct {
 	PrivateKeyPath string
 	KnownHostsPath string
 }
+
+// AutoConfigureOpenSSH adds an Include directive to the default user OpenSSH
+// config file (~/.ssh/config) to include the vnet_ssh_config file found under
+// profilePath.
+func AutoConfigureOpenSSH(ctx context.Context, profilePath string, overrideUserSSHConfigPath ...string) (err error) {
```

Comment:
I don't love the idea of parameter used only in tests but couldn't figure out a better way. 
We should at least mention in the godoc what it does and why (or maybe even change its name to make sure it won't be used outside of tests?)